mbzirc_custom/mbzirc_naive_radar/src/NaiveRadar.cc

```diff
@@ -0,0 +1,233 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include <ignition/msgs.hh>
+#include <ignition/plugin/Register.hh>
+
+#include <ignition/sensors/Noise.hh>
+
+#include <ignition/gazebo/Util.hh>
+#include <ignition/gazebo/components/Model.hh>
+#include <ignition/gazebo/components/Pose.hh>
+#include <ignition/gazebo/components/ParentEntity.hh>
+#include <ignition/gazebo/EntityComponentManager.hh>
+
+#include "NaiveRadar.hh"
+
+using namespace mbzirc;
+
+/////////////////////////////////////////////////
+NaiveRadar::NaiveRadar()
+{
+}
+
+/////////////////////////////////////////////////
+NaiveRadar::~NaiveRadar() = default;
+
+//////////////////////////////////////////////////
+void NaiveRadar::Configure(const ignition::gazebo::Entity &_entity,
+    const std::shared_ptr<const sdf::Element> &_sdf,
+    ignition::gazebo::EntityComponentManager &_ecm,
+    ignition::gazebo::EventManager &/*_eventMgr*/)
+{
+  // parse configuration parameters from SDF
+  auto sdf = const_cast<sdf::Element *>(_sdf.get());
```

Comment:
Why was this `const_cast` necessary?

Reply:
nope removed. [90641cf](https://github.com/osrf/mbzirc/pull/105/commits/90641cfbd946be5ac560a5358e8ddc659b503584).

Reply:
urgh spoke and merged too soon. The cast is needed. Fixed in [49794e4](https://github.com/osrf/mbzirc/pull/110/commits/49794e4c0be7050d487a0fc8dcb6cf3a1ac5e68f) in #110